packages/rancher-istio/rancher-istio/charts/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: 1.19 - 1.22
+  catalog.cattle.io/kube-version: 1.20 - 1.23
```

Comment:
nit: please change the kube-version annotation to `'>= 1.20.0-0 < 1.24.0-0'`, and rancher-version to `'>= 2.6.0-0 <= 2.6.100-0'`. We are making an effort to stick to one format so the more charts we can switch over the better.

Reply:
@PennyScissors Should the kube-version annotation be `'>= 1.20.0-0 < 1.24.0-0'`? Since istio 1.13 does not support k8s 1.24 yet.